conan/cli/commands/editable.py

```diff
@@ -21,55 +20,56 @@ def editable_add(conan_api, parser, subparser, *args):
     package is required, it is used from this <path> location instead of from the cache
     """
     subparser.add_argument('path', help='Path to the package folder in the user workspace')
-    subparser.add_argument('reference', help='Package reference e.g.: mylib/1.X@user/channel')
+    add_reference_args(subparser)
     subparser.add_argument("-of", "--output-folder",
                            help='The root output folder for generated and build files')
```

Comment:
Is this `--output-folder` still relevant?

Reply:
Yes, it is passed below as ``args.output_folder`` and managed by editables.